proposals/dao/tip_115.ts

```diff
@@ -47,6 +52,9 @@ const validate: ValidateUpgradeFunc = async (addresses, oldContracts, contracts,
   expect(await contracts.rariTimelockFeiOldLens.balanceReportedIn()).to.be.equal(addresses.fei);
   expect(await contracts.tribalCouncilTimelockFeiLens.balanceReportedIn()).to.be.equal(addresses.fei);
   expect(await contracts.namedStaticPCVDepositWrapper.numDeposits()).to.be.equal('0');
+  expect(await contracts.ethToDaiLBPSwapper.swapEndTime()).to.be.gt(
+    ethers.BigNumber.from((await time.latest()).toString())
+  );
```

Comment:
Depends how confident are, but I would probably make sure you can do a swap via the LBP pool here